azerothcore/azerothcore-wotlk
# fix(Core): Compilation with EXTRA_LOGS in Windows

## Changes Proposed:
-  There is an issue when you set the EXTRA_LOGS at CMake not allowing to compile as intended, so with this fix it compiles without issues.


## Issues Addressed:
- If you enable at CMake the configs ENABLE_EXTRA_LOGS and ENABLE_EXTRAS in Windows, the compilation fails.


## Tests Performed:
- Just the build and it passed


## How to Test the Changes:
 - Just open CMAKE, set ENABLE_EXTRAS/ENABLE_EXTRAS_LOG to enabled (the checkmark), and then try to compile, before this small fix, it does not finish, after the fix, it should pass.



## Target Branch(es):



 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
